use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,238 @@
+   ```
+
+   Additionally, you should be able to access the device node for the graphics device:
+
+   ```
+   $ ls /dev/dri
+     card0  renderD128
+   ```
+
+## GVT-g with Kata Containers
+
+Follow  the steps below to pass an Intel Graphics device in GVT-d mode to a Kata Container:
+
+1. Find the Bus-Device-Function(BDF) for GPU device:
+
+   ```
+   $ sudo lspci -nn -D | grep Graphics
+     0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Broadwell-U Integrated Graphics [8086:1616] (rev 09)
+   ```
+
+   Run the above command to find out the BDF for the GPU device on host.
+   From the above output, PCI addres "0000:00:02.0" is assigned to the GPU device.
+   This BDF will be used later to unbind the GPU device from host.
+
+2. Choose the MDEV type for VGPU:
+
+   For background on dev types, please follow this [kernel documentation](https://github.com/torvalds/linux/blob/master/Documentation/vfio-mediated-device.txt)
+
+   * List out the mdev types for the VGPU:
+
+     ```
+     $ BDF="0000:00:02.0"
+
+     $ ls /sys/devices/pci0000:00/$BDF/mdev_supported_types
+       i915-GVTg_V4_1  i915-GVTg_V4_2  i915-GVTg_V4_4  i915-GVTg_V4_8
+     ```
+
+   * Inspect the mdev types and choose one that fits your requirement:
+
+     ```
+     $ cd /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4_8 && ls
+       available_instances  create  description  device_api  devices
+
+     $ cat description
+       low_gm_size: 64MB
+       high_gm_size: 384MB
+       fence: 4
+       resolution: 1024x768
+       weight: 2
+
+     $ cat available_instances
+       7
+     ```
+
+     The output of file description represents the GPU resources that will be assigned to the VGPU with specified MDEV type.<br/>
+     The output of file available_instance represents the remaining amount of VGPU can be created with specified MDEV type.
+
+3. Create a VGPU:
+
+   ```
+   $ # Generate a uuid
+   $ uuid   
+     bbc4aafe-5807-11e8-a43e-03533cceae7d
+   $ gpu_uuid="bbc4aafe-5807-11e8-a43e-03533cceae7d"
+
+   $ # Write the uuid to the `create` file under the chosen mdev type
+   $ sudo su
+   # echo $(gpu_uuid) > /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4/create
+   ```
+
+   The above steps create a VGPU.
+
+4. Find the IOMMU group for the VGPU:
+
+   ```
+   $ ls -la  /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4/devices/$(gpu_uuid)/iommu_group
+     lrwxrwxrwx 1 root root 0 May 18 14:35 devices/bbc4aafe-5807-11e8-a43e-03533cceae7d/iommu_group -> ../../../../kernel/iommu_groups/0
+
+   $ ls /dev/vfio
+     0 vfio
+   ```
+
+   From the above output, IOMMU group "0" is created.<br/>
+   The device node "/dev/vfio/0" can now be used in docker command line to pass
+   the VGPU to a Kata Container.
+
+5. Start Kata container with GPU device enabled:
+
+   ```
+   $ docker run -it --runtime=kata-runtime --rm  --device /dev/vfio/0  -v /dev:/dev debian /bin/bash
```

Comment:
For consistency, and to avoid issues where the user isn't in the correct group, could you prefix with `sudo` here?

Reply:
fixed.